### ISSUE TITLE
Rename Bevy Assets page

### ISSUE THREAD

**Issue by alice-i-cecile** [bevyengine/bevy-website#175]
https://bevyengine.org/assets/  is very useful, but "Assets" does not reflect its contents. I would expect "asset pack", rather than "general Bevy resources". This makes it hard to link to coherently and hinders discoverability.

**Comment by cart** [comment]
Relevant discord thread with some context: https://discord.com/channels/691052431525675048/695741366520512563/852262367520751626

**Comment by alice-i-cecile** [comment]
I'm running up against this again when writing docs. It doesn't make any sense to call it "Bevy Assets" when "assets" is already a reserved word in Bevy for something completely different. I have no idea how to properly frame this when writing. For example:

> Bevy's modular nature tends to result in simple plug-and-play interoperability and easy extensibility, so don't be afraid to try out plugins that seem interesting or useful for your game.
> 1. Find a Bevy plugin (such as from our [collection of assets](https://bevyengine.org/assets/)).

This makes very little sense to users coming from a programming background, where source code is never referred to as "assets".

This is going to repeatedly confuse users and hurt searchability; I'd really prefer to change this before we launch 0.6. My stance is that users coming from other game engines will be able to quickly figure out what our analogue to e.g. Unity Asset Store, and Unreal and [Godot's](https://godotmarketplace.com/) Marketplace just fine; it's new users who we need to communicate with clearly.

Both "X Asset Store" and "X Marketplace" are much more clear than just "Assets": they communicate the commercial / bazaar intent, rather than just "here's a collection of things". Marketplace is particularly good, because it doesn't tie us to assets in such a strong way.

I would be perfectly happy with "Bevy Marketplace", even if for now it's decentralized and entirely free of charge. "Bevy Bazaar" is a fun alliterative alternative that captures the chaos a bit better, although it's a bit harder to search for / less standard.

**Comment by Nilirad** [comment]
I think "Packages" describes well what we're referring to.

Definition of package, from dictionary.com:

> a bundle of something, usually of small or medium size, that is packed and wrapped or boxed

**Comment by cart** [comment]
Yeah I've been going back and forth on this a lot too. Ultimately, I'm willing to consider Bevy Marketplace, but it feels odd to call it that when you can't sell anything. A Bazaar is also a marketplace, so that suffers from the same problem imo. While everything is free,  Bevy Library seems like a reasonable fit, but we might want to rename it if/when we enabling selling things.

Its worth calling out that the big open-source engines in this category refers to this "thing" as Asset

![image](https://user-images.githubusercontent.com/2694663/128768125-7e64ace4-2fa9-44a8-92e1-e475dc768d6a.png)
![image](https://user-images.githubusercontent.com/2694663/128768180-c8087028-cd02-44d6-91a4-f7c6ed3ba598.png)

Bevy feels right at home in that context imo:
![image](https://user-images.githubusercontent.com/2694663/128768254-f96b4b9a-fe69-43fa-b14d-d12e740d2563.png)

Then theres also Unity:
![image](https://user-images.githubusercontent.com/2694663/128768313-406852f8-e661-4db2-9979-7815f9ed2abb.png)

I personally think we should scope the discussion down to Bevy Assets,  Bevy Asset Library, Bevy Asset Store, Bevy Marketplace, and Bevy Library

I think I agree with your idea of adding a qualifier to Assets, like "Asset Store", "Asset Portal", "Asset Library". Godot does this ("Assets" in the headerbar, "Asset Library" as the long-form name).

I think my preference is "Asset Library" (while everything is free), then we could rename it to "Asset Store" if/when we add the ability to sell things in it. The website header bar would still just be "Assets" (for terseness), but when referring to it as a concept, we would call it the "Asset Library".

**Comment by alice-i-cecile** [comment]
>I' m willing to consider Bevy Marketplace, but it feels odd to call it that when you can't sell anything.

As an data point, VS Code calls its "place where you find extensions" a marketplace as well, although I had no idea you even *could* buy extensions there (TIL!).

>  The website header bar would still just be "Assets" (for terseness),

I'd really prefer to avoid doing this. It's confusing and looks unpolished if the link labelled "Bevy Asset Library" is to what is apparently "Bevy Assets".

> Its worth calling out that the big open-source engines in this category refers to this "thing" as Asset

They don't have to deal with the name-space conflict though :( Their "assets" (images, sounds, etc.) are called [resources](https://docs.godotengine.org/en/3.0/getting_started/step_by_step/resources.html). It's fundamentally confusing to find plugins / code in the Assets page

> I personally think we should scope the discussion down to Bevy Assets, Bevy Asset Library, Bevy Asset Store, Bevy Marketplace, and Bevy Library

I think that's reasonable. Marketplace is more consistent than Bazaar, and otherwise has similar strengths.

My preference would be for Bevy Library of the choices listed. Bevy Marketplace is also good, to avoid having to rename later and to improve consistency with the others in the space.

**Comment by alice-i-cecile** [comment]
Update from this: `Bevy Marketplace` is the tentative agreed upon solution following discussions on Github, but we won't migrate until you can actually buy something.
